pom.xml

```diff
@@ -175,6 +189,36 @@
                 </configuration>
             </plugin>
 
+            <!-- Generate REST default interface from Swagger spec -->
+            <plugin>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>${swagger.codegen.version}</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
```

Comment:
Probably add apiPackage and modelPackage option so the generated code won't be under io.swagger.